packages/context/src/binding-filter.ts

```diff
@@ -125,7 +133,26 @@ export function isBindingTagFilter(
  * ctx.findByTag({controller: ANY_TAG_VALUE})
  * ```
  */
-export const ANY_TAG_VALUE = Symbol.for('loopback.AnyTagValue');
+export const ANY_TAG_VALUE: TagValueMatcher = (tagMap, tagName) =>
+  tagName in tagMap;
+
+/**
+ * Create a tag value matcher function that returns `true` if the target tag
+ * value is an array that includes the given item value.
+ * @param itemValue - Tag item value
+ */
+export function includeTagValueMatcher(itemValue: unknown): TagValueMatcher {
```

Comment:
I find this function name difficult to read and understand what the function is doing. Take a look at an example use in your tests:

```ts
      const filter = filterByTag({
         controller: includeTagValueMatcher('MyController'),
         name: 'my-name',
       });
```

Let's find a better name please. How about `containsTagValue` or `includesTagValue`?

```ts
      const filter = filterByTag({
         controller: containsTagValue('MyController'),
         name: 'my-name',
       });
```

In general, I find the overall design of this feature difficult to comprehend too. `TagValueMatcher` is accepting a tag map and a tag name, where the map is coming from the binding (which is expected), but the name is supplied by some intermediate parts (code inside `filterByTag` in this case). When reading the example above, we need to understand a lot about the way how LoopBack works with binding tags.

I understand the current `TagValueMatcher` offers a lot of flexibility, but the cost is too high in my opinion.

Have you considered a simpler design where the matcher receives just the tag value to match?

```ts
export type TagValueMatcher = (value: unknown) => boolean;
```

Then the "includes" matcher can be greatly simplified or even not needed at all:

```ts
      const filter = filterByTag({
         controller: value => 
           value === 'MyController' || 
           (Array.isArray(value) && value.includes('MyController')),
         name: 'my-name',
       });
```

Reply:
Sounds good. I now use a similar syntax as https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach to pass in the following params:

`(tagValue, tagName, tagMap)`